### ISSUE TITLE
Download Limits

### ISSUE THREAD

**Issue by junio-AC** [JBGruber/traktok#3]
Hi,
I'm once again here to ask some questions about the script and once again I want to thank you for the generous work done :)

My questions are related to the downloads limit, specifically:

1) Is there a way to download more than the last 30 URL videos from a profile?

2) In the hashtag download function it seems I get a partial download of them. Just as an example), the code returns me "23700 videos found for #HASHTAG" but, in the end, I get 300. Is that the limit? Is there a way to get all of them?

Thank you so much!

EDIT
Sorry, just found out the URL limit has been already addressed in this issue (https://github.com/JBGruber/traktok/issues/1), hopefully, a solution will eventually arrive :)

**Comment by JBGruber** [comment]
1. I can't fix that at the moment, as you've already discovered. Happy to give it a try, but I don't have time to do the research on encryption in R.
2. There was a bug in the function that made it fail when even one field in the returned json was missing. I just fixed this (just reinstall the package to update). However, I just tested your example and I still do not get the 2 900 000 videos it claims to have found. After about 8 000 videos, TikTok returns the status 
```
> res$hasMore
[1] FALSE
```
In other words, there is no more to return for the search request. 

This could have two reasons: a. the initial number of 2 900 000 videos that TikTok reports is wrong; b. TikTok only lets you scroll down that far. In both cases there seems little that I can do.

**Comment by junio-AC** [comment]
Thank you so much for your reply! the actual hashtag I was looking for was #elezioni22 but I guess the outcome does not change.

**Comment by JBGruber** [comment]
Out of scope for now
